packages/web-ui/README.md

```diff
@@ -1,46 +1,33 @@
-# Getting Started with Create React App
+# Web UI
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Try out `abi-to-sol` via a web UI.
 
-## Available Scripts
+## Getting started
 
-In the project directory, you can run:
+### Prerequisites
 
-### `yarn start`
+- Node 16
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Instructions
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+1. If dependencies aren't already installed, run the following in the **root** of the repo:
 
-### `yarn test`
+```bash
+$ yarn
+```
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+2. If `abi-to-sol` hasn't been built, i.e. `packages/abi-to-sol/dist` doesn't exist, run the following in `packages/abi-to-sol`:
 
-### `yarn build`
+```bash
+$ yarn prepare
+```
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Troubleshooting
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### `Error: error:0308010C:digital envelope routines::unsupported`
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Switch to Node v16. Explanation of the issue can be found [here](https://stackoverflow.com/a/73027407/6475944).
 
-### `yarn eject`
+### `Cannot find module or its corresponding type declarations.` for `abi-to-sol`
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+`packages/abi-to-sol` hasn't been built. Go to `packages/abi-to-sol` and run `yarn prepare`.
```

Comment:
Hm kind of annoying that `yarn` doesn't automatically build everything in the repo root. Nonetheless, good to have these instructions here!